tcms/utils/permissions.py

```diff
@@ -3,30 +3,44 @@
 from django.contrib.auth.models import Group, Permission
 
 
-def assign_default_group_permissions():
+def assign_default_group_permissions(output=None, refresh_all=False):
     """
         Adds the default permissions for Administrator and Tester
         groups!
     """
     admin = Group.objects.get(name='Administrator')
-    if admin.permissions.count() == 0:
+    if output:
+        output.write('\nAdding permissions to Administrator group:')
```

Comment:
This message is redundant b/c below you have a more detailed message for each permission being added. Remove it.